src/base/torrentfilter.cpp

```diff
@@ -50,15 +50,17 @@ const TorrentFilter TorrentFilter::ErroredTorrent(TorrentFilter::Errored);
 
 using BitTorrent::Torrent;
 
-TorrentFilter::TorrentFilter(const Type type, const TorrentIDSet &idSet, const QString &category, const QString &tag)
+TorrentFilter::TorrentFilter(const Type type, const std::optional<TorrentIDSet> &idSet
+                             , const std::optional<QString> &category, const std::optional<QString> &tag)
```

Comment:
If we don't have a rule (official or not) I would prefer the comma to be right after the paremeter name on the same line.

Reply:
> If we don't have a rule (official or not) I would prefer the comma to be right after the paremeter name on the same line.

We have been guided by this (unofficial) rule for quite a long time.